src/main/java/org/sciborgs1155/robot/Robot.java

```diff
@@ -63,22 +66,27 @@ private void configureGameBehavior() {
    * Configures subsystem default commands. Default commands are scheduled when no other command is
    * running on a subsystem.
    */
-  private void configureSubsystemDefaults() {
+  // private void configureSubsystemDefaults() {
+  //   drive.setDefaultCommand(
+  //       drive
+  //           .drive(() -> -driver.getLeftY(), () -> -driver.getLeftX(), () -> -driver.getRightX())
+  //           .withName("teleop driving"));
+  // }
+
+  private void configureJoysticks() {
     drive.setDefaultCommand(
-        drive
-            .drive(() -> -driver.getLeftY(), () -> -driver.getLeftX(), () -> -driver.getRightX())
-            .withName("teleop driving"));
+        new RunCommand(()-> drive.setVoltage(joystickL.getY(), joystickR.getY())));
```

Comment:
Several things:
1. For aesthetic reasons, you should probably use `Commands.run` instead of `new RunCommand`, since the `Commands` factories are nicer to read
2. This method should be named `configureSubsystemDefaults` to be consistent with the other naming
3. `drive.setVoltage` is not a good idea to directly input joystick values into. You should instead scale the joystick values (between -1 and 1) and use them with WPILib's `DifferentialDrive` helper methods.